flow360/component/simulation/operating_condition.py

```diff
@@ -4,47 +4,24 @@
 
 from flow360.component.simulation.framework.base_model import Flow360BaseModel
 
-"""
-    Defines all the operating conditions for different physics.
-    The type of operating condition has to match its volume type.
-    Operating conditions defines:
-    1. The physical (non-geometrical) reference values for the problem.
-    2. The initial condition for the problem.
-
-    TODO:
-    1. What other types of operation conditions do we need?
-"""
-
-
-class TurbulenceQuantities(Flow360BaseModel):
-    """PLACE HOLDER, Should be exactly the the same as `TurbulenceQuantitiesType` in current Flow360Params"""
-
-    pass
 
+class DensityTemperature(Flow360BaseModel):
+    temperature: TemperatureType = pd.Field(288.15)
+    density: DensityType = pd.Field(1.225)
 
-class ExternalFlowOperatingConditions(Flow360BaseModel):
-    Mach: float = pd.Field()
-    alpha: float = pd.Field(0)
-    beta: float = pd.Field(0)
-    temperature: float = pd.Field(288.15)
-    reference_velocity: Optional[float] = pd.Field()  # See U_{ref} definition in our documentation
 
-    initial_flow_condition: Optional[tuple[str, str, str, str, str]] = pd.Field(
-        ("NotImplemented", "NotImplemented", "NotImplemented")
-    )
-    turbulence_quantities = Optional[TurbulenceQuantities] = pd.Field()
+class StandardAtmosphere(Flow360BaseModel):
+    altitude: LengthType = pd.Field(0)
```

Comment:
maybe altitude and temperature can be another set of combination (if valid)? I thought standard atmosphere condition is a const condition for example the altitude is just 0?

Reply:
standard atmosphere is a model that relates the altitude to pressure, density and temperature.

Reply:
Right. It is such model: https://www.digitaldutch.com/atmoscalc/

we have it already implemented for panel